observatory-platform/observatory/platform/observatory_config.py

```diff
@@ -896,10 +896,14 @@ def make_airflow_variables(self) -> List[AirflowVariable]:
         """
 
         # Create airflow variables from fixed config file values
-        variables = [AirflowVariable(AirflowVars.ENVIRONMENT, self.backend.environment.value)]
+        data_path = "/opt/observatory/data"  # Hardcoded in docker-compose.observatory.yml.jinja2
```

Comment:
I think you could not add `data_path` as a cloud secret because it is always set from an environment variable (as hardcoded in the Docker Compose file.

You could also remove these two lines from the Docker Compose file now: https://github.com/The-Academic-Observatory/observatory-platform/blob/develop/observatory-platform/observatory/platform/docker/docker-compose.observatory.yml.jinja2#L89-L90